examples/vms/eln_data_vms.yaml

```diff
@@ -44,10 +42,9 @@ instrument:
         identifier: null
     energydispersion:
       scheme: hemispherical
-      entrance_slit: unknown
       energy_scan_mode: fixed_analyser_transmission
       diameter:
-        unit: mm
+        unit: milli-m

```

Comment:
is this compatible with nomad? Nomad accept `mm`.

Reply:
So far, these units have worked. There will be unit validation soon, then I may refactor this part.